src/main/scala/io/scalac/tezos/translator/model/SendEmail.scala

```diff
@@ -37,24 +36,31 @@ object SendEmail {
     new SendEmail(uid, emailAddress, subject, message) {}
   }
 
-  def fromSendEmailRoutesDto(dto: SendEmailRoutesDto, adminEmail: EmailAddress): SendEmail =
-    new SendEmail(
-      uid = Uid(),
-      to = adminEmail,
-      subject = "Contact request",
-      content = ContactFormContent(
-        name = dto.name,
-        phone = dto.phone,
-        email = dto.email,
-        content = dto.content
-      )
-    ) {}
+  def fromSendEmailRoutesDto(dto: SendEmailRoutesDto, adminEmail: EmailAddress): Try[SendEmail] = {
+    for {
+      email   <-   dto.email match {
```

Comment:
I know its a matter of preference, but I prefer to have such code formatted in this way:
```
for {
  email   <-  dto.email match {
                case Some(e) => EmailAddress.fromString(e).map(Some(_))
                case None => Success(None)
              }
  contact <-  Contact.tryToCreateContact(dto.phone, email)
} yield ...
```
I find it way more readable.
Could you please change this ?

Reply:
I like your preference so I can change mine  😄